### PR TITLE
prep-css-box-model.

### PR DESCRIPTION
**Pull request by Timothy-Buerger**
prep-css-box-model

### CODE REVIEW

File: prep-css-box-model/style.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  margin: 20px;
+  padding: 10px;
+  border-style: double;
+  border width: 3px;
```
Comment (scbowler) [inline prep-css-box-model/style.css]: This should be `border-width`

File: prep-css-box-model/style.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  border color: darkgray;color
```
Comment (scbowler) [inline prep-css-box-model/style.css]: This should be `border-color` and the color should only be `darkgray;`, you don't need the word `color` after. The full rule should look like:

```CSS
border-color: darkgray;
```

**Review by scbowler — APPROVED** [review]
On the right track, but a couple updates needed. Let me know if you have any questions.

**Review by scbowler — CHANGES_REQUESTED** [review]
Changes needed, didn't mean to approve

**Comment by Timothy-Buerger** [comment]
prep-css-box-model

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**